test/forms/mappings/SelectLocalDateFormatterSpec.scala

```diff
@@ -35,29 +35,42 @@ class SelectLocalDateFormatterSpec extends SpecBase {
       )
     }
 
+    "return the correct FormError with key when the supplied data has empty month and year" in new SetUp {
+      localDateFormatterWithEmptyMonthAndYear.bind(key, bindDataDateWithEmptyMonthAndYear) shouldBe
+        Left(Seq(FormError("start.month", List(emptyMonthAndYearMsgKey), List())))
+    }
+
     "return the correct FormError with keys when the supplied data is empty month" in new SetUp {
       localDateFormatter.bind(key, bindDataDateWithEmptyMonth) shouldBe
-        Left(Seq(FormError("start.month", List(invalidMsgKey), List())))
+        Left(Seq(FormError("start.month", List(emptyMonthMsgKey), List())))
     }
 
     "return the correct FormError with keys when the supplied data empty year" in new SetUp {
       localDateFormatter.bind(key, bindDataDateWithEmptyYear) shouldBe
-        Left(Seq(FormError("start.year", List(invalidMsgKey), List())))
+        Left(Seq(FormError("start.year", List(emptyYearMsgKey), List())))
     }
 
     "return the correct FormError with keys when the supplied data is invalid month" in new SetUp {
 
```

Comment:
Could we remove this space here please, to be in align with other tests, i know you haven't added this

Reply:
Done